sky/engine/core/painting/Paint.dart

```diff
@@ -27,54 +27,106 @@ enum StrokeCap {
 class Paint {
   /// Whether to paint inside shapes, the edges of shapes, or both.
   ///
-  /// If null, defaults to [PaintingStyle.fill].
-  PaintingStyle style;
+  /// Defaults to [PaintingStyle.fill].
+  PaintingStyle style = _kDefaultStyle;
+  static const PaintingStyle _kDefaultStyle = PaintingStyle.fill;
 
   /// How wide to make edges drawn when [style] is set to
   /// [PaintingStyle.stroke] or [PaintingStyle.strokeAndFill]. The
   /// width is given in logical pixels measured in the direction
   /// orthogonal to the direction of the path.
   ///
-  /// The values null and 0.0 correspond to a hairline width.
-  double strokeWidth;
+  /// Defaults to 0.0, which corresponds to a hairline width.
+  double strokeWidth = _kDefaultStrokeWidth;
+  static const double _kDefaultStrokeWidth = 0.0;
 
   /// The kind of finish to place on the end of lines drawn when
   /// [style] is set to [PaintingStyle.stroke] or
   /// [PaintingStyle.strokeAndFill].
   ///
-  /// If null, defaults to [StrokeCap.butt], i.e. no caps.
-  StrokeCap strokeCap;
+  /// Defaults to [StrokeCap.butt], i.e. no caps.
+  StrokeCap strokeCap = _kDefaultStrokeCap;
+  static const StrokeCap _kDefaultStrokeCap = StrokeCap.butt;
```

Comment:
IMHO, we should skip these defaults.  The null values mean we don't have to call into skia in C++, which means we get the default in Skia.  That's going to be slightly more efficient.